WISE-Community/WISE
# HibernateRunDaoTest with Spring-Boot configuration

It also uses H2 in-memory database.

How to test:
1. Run maven test. 184 test should run and pass. 
2. Open HibernateRunDaoTest.java in vscode. There should be a "Run | Debug" link above the class definition if you have installed Java Debugger plugin. Click on Run. All 5 tests in the class should run and pass.
3. Try running or debugging few of the test methods. They should pass.

Closes #2065
